lib/elven_gard/helpers/frontend.ex

```diff
@@ -122,7 +126,7 @@ defmodule ElvenGard.Helpers.Frontend do
 
           x ->
             raise """
-            #{unquote(handler)}.handle_packet/2 have to return `{:cont, client}`, \
+            #{unquote(handler)}.handle_packet/3 have to return `{:cont, client}`, \
```

Comment:
Should it be `handle_message` instead ?

Reply:
Why `handle_message` ?

Reply:
i can't find any function named `handle_packet/3`

Reply:
https://github.com/ImNotAVirus/ElvenGard_V2/blob/7d125c316b3d2261dddb284db3046c495595ca05/lib/elven_gard/helpers/frontend.ex#L160

Reply:
oh nvm